Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```diff
@@ -97,6 +97,13 @@
     walkModifier: 0.95
     sprintModifier: 0.9
     enabled: false
+  - type: Appearance
+  - type: GenericVisualizer
+    visuals:
+      enum.ToggleVisuals.Toggled:
+        enum.ToggleVisuals.Layer:
+          True: {state: icon-on}
+          False: {state: icon}
```

Comment:
This doesn't seem to fix anything. The core of the problem is in `Content.Client/JetpackSystem`. The case is that Jetpack and magboots use the same postfix `-on`, same with item icon.
Besides, `- type: Appearance` already inherited by the parent, so no need to duplicate it.

Reply:
My bad, I seem to have not gotten a notification over this reply. I am very new to the codebase but when I was poking around with this, I noticed that that was one difference between the boots and, upon duplicating it it seemed to work. You're saying that these lines don't do anything at all? If so, I could just remove that file from the PR...